homeassistant/components/notify/demo.py

```diff
@@ -22,6 +22,11 @@ def __init__(self, hass):
         """Initialize the service."""
         self.hass = hass
 
+    @property
+    def targets(self):
+        """Return a dictionary of registered targets."""
+        return {"test target": {}}
```

Comment:
Let's keep this a list / iterable that returns string targets. So just return `["test target"]`. No need to expose more data